src/Symfony/Component/HttpClient/Response/ResponseTrait.php

```diff
@@ -49,9 +49,10 @@ trait ResponseTrait
     private $content;
 
     private $info = [
-        'response_headers' => [],
```

Comment:
let's keep the order :)

Reply:
Ok, same for the interface? Or is alphabetically fine there? :)

Reply:
For the interface it's ok: there is not concern related to merges.

Reply:
Ah, the merge issue. I see. There you go :)